amazon-connect/callstats-amazon-connect-shim.js

```diff
@@ -195,22 +200,42 @@
           CallstatsJabraShim.stopJabraMonitoring();
         }
         if (enableVoiceActivityDetection) {
-          localAudioAnalyser.stop();
-          remoteAudioAnalyser.stop();
+          if (localAudioAnalyser) {
+            localAudioAnalyser.stop();
+          }
+          if (remoteAudioAnalyser) {
+            remoteAudioAnalyser.stop();
+          }
           if (eventList.length > 0) {
             CallstatsAmazonShim.callstats.sendCustomEvent(null, 
               confId, eventList);
             eventList = [];
           }
         }
+        if (!isConferenceSummarySent) {
+          var event = {
+            type: 'conferenceSummary',
+            timestamp: getTimestamp(),
+            pstnTime: pstnTime,
+            ringingTime: ringingTime,
+            contactID: confId,
+            isCallForwarded: isCallForwarded,
+          }
+          CallstatsAmazonShim.callstats.sendCustomEvent(null, confId, [event]);
```

Comment:
Can we send it as the normal event? Sending it as the custom event complicates things significantly in the stream analyzer.

Reply:
@marcinag It is sent as normal event, Instead of creating a new API in callstats.js we are making use of it. This event will be stored in the conference events.